### DISCUSSION TITLE
Filament v3.2 ExportAction -  filter data to export

### DISCUSSION THREAD

**Discussion by carloslopes34** [filamentphp/filament#11510] [Help] [answered]
### Package

Actions

### Package Version

v3.2

### How can we help you?

How to export the data using a filter by date? Is it possible to have this option selectable by the user?

**Comment by zeeshantariq08** [comment, marked as answer]
have you tried this package?
https://github.com/alperenersoy/filament-export

**Reply by carloslopes34** [reply]
tks @zeeshantariq08  this resolve my problem.
